src/fmeval/eval_algorithms/prompt_stereotyping.py

```diff
@@ -13,29 +12,73 @@
     EvalAlgorithm,
     EvalOutput,
     EvalScore,
-    EVAL_DATASETS,
-    DATASET_CONFIGS,
     get_default_prompt_template,
 )
 from fmeval.eval_algorithms.util import (
     aggregate_evaluation_scores,
     validate_dataset,
-    generate_model_predict_response_for_dataset,
-    generate_prompt_column_for_dataset,
     generate_output_dataset_path,
     save_dataset,
+    get_dataset_configs,
 )
 from fmeval.model_runners.model_runner import ModelRunner
 from fmeval.perf_util import timed_block
+from fmeval.transforms.common import GeneratePrompt, GetLogProbabilities
+from fmeval.transforms.transform import Transform
+from fmeval.transforms.transform_pipeline import TransformPipeline
 
 LOG_PROBABILITY_DIFFERENCE = "log_probability_difference"
 PROMPT_STEREOTYPING = EvalAlgorithm.PROMPT_STEREOTYPING.value
 logger = logging.getLogger(__name__)
 
 
+class PromptStereotypingScores(Transform):
```

Comment:
NTH: this is in fact a general operation that you can do to compare any two passages via the probability assigned by the model. As such this is not necessarily tied to "stereotyping". So would suggest to recall this as ProbabilityRatio or something along the line